package.json

```diff
@@ -1,69 +1,43 @@
 {
   "name": "fbw-installer",
   "productName": "FlyByWire Installer",
-  "version": "1.3.0-dev",
+  "version": "2.0.0-dev17",
```

Comment:
Make sure to change this before merging

Reply:
Nah that's fine, merge doesn't mean it gets released in that state. I'll create a separate branch for the actual release

Reply:
Discussed in discord, resolving